app/controllers/api/api_controller.rb

```diff
@@ -5,15 +5,15 @@ module Api
 
   class ApiController < ApplicationController
     include JsonApiController
-    include RoleControl::RoledController
+    include JsonApiController::CheckResourcesExist
```

Comment:
Is the plan to move this to each controller as i proposed in #2688 instead of being less visible here in the API controller?

Reply:
Yes